apps/web/lib/settings/left-side-setting-menu.tsx

```diff
@@ -146,14 +141,9 @@ export const LeftSideSettingMenu = ({ className }: { className?: string }) => {
 								? ' text-[#3826a6] text-primary font-semibold'
 								: ' border-l-transparent font-normal dark:text-[#7E7991]'
 						}`}
-						wrapperClassName={`w-full border-t-0 border-r-0 border-b-0 rounded-none
-						font-normal text-[#7e7991] justify-start text-sm pt-[24px] pb-[24px] pl-[24px]
-	border-l-[5px] ${
-		activePage === '/settings/team'
```

Comment:
@Cedric921 can you check this PR, why we had this conditional styling and now it's removed?

Reply:
And since these are the same components as the previous one, I just adapted the rest.